resources/public/api/conf/2.0/schemas/retrieve_foreign_property_bsas.json

```
@@ -0,0 +1,209 @@
+              "example": "123.12"
+            }
+          }
+        },
+        "expensesBreakdown": {
+          "type": "object",
+          "description": "Total expenses broken down by type for the tax year to date",
+          "properties": {
+            "premisesRunningCosts": {
+              "type": "number",
+              "description": "Rent, rates, insurance, ground rents etc. The value must be between 0 and 99999999999.99.",
+              "example": "123.12"
+            },
+            "repairsAndMaintenance": {
+              "type": "number",
+              "description": "Property repairs and maintenance. The value must be between 0 and 99999999999.99.",
+              "example": "123.12"
+            },
+            "financialCosts": {
+              "type": "number",
+              "description": "Loan interest and other financial costs. The value must be between 0 and 99999999999.99.",
+              "example": "123.12"
+            },
+            "professionalFees": {
+              "type": "number",
+              "description": "Legal, management and other professional fees. The value must be between 0 and 99999999999.99.",
+              "example": "123.12"
+            },
+            "travelCosts": {
+              "type": "number",
+              "description": "Car, van and travel costs incurred in running a property business. The value must be between 0 and 99999999999.99.",
+              "example": "123.12"
+            },
+            "costOfServices": {
+              "type": "number",
+              "description": "Cost of services provided, including wages. The value must be between 0 and 99999999999.99.",
+              "example": "123.12"
+            },
+            "residentialFinancialCost": {
+              "type": "number",
+              "description": "Captures residential financial cost that can be deductible from rental income (tax relief). The value must be between 0 and 99999999999.99.",
+              "example": "123.12"
+            },
+            "broughtFwdResidentialFinancialCost": {
+              "type": "number",
+              "description": "Amount of relief brought forward for restricted residential financial costs. The value must be between 0 and 99999999999.99.",
+              "example": "123.12"
+            },
+            "other": {
+              "type": "number",
+              "description": "Other allowable property expenses. The value must be between 0 and 99999999999.99.",
+              "example": "123.12"
+            },
+            "consolidatedExpenses": {
+              "type": "number",
+              "description": "Sum of all expenses for the specified period. The value must be between 0 and 99999999999.99.",
+              "example": "123.12"
+            }
+          }
+        }
+      }
```

`total` is required.

```suggestion
      },
      "required": [
        "total"
      ]
```